### ISSUE TITLE
Wrong types definition

### ISSUE THREAD

**Issue by tr3v3r** [maxkomarychev/react-native-ultimate-config#3]
I'm using React Native with TypeScript. So, after :
```
import config from 'react-native-ultimate-config'
```
I tried to use config and got ts error: 
```
'config' only refers to a type, but is being used as a value here.
```

So I'd like to propose you instead of generating `index.d.ts` file like:

```
// DO NOT COMMIT OR EDIT THIS FILE
declare module 'react-native-ultimate-config' {
    export interface ConfigVariables {
        TEST: string
    }
    export default ConfigVariables;
}
```

Do:
```
// DO NOT COMMIT OR EDIT THIS FILE
interface ConfigVariables {
   TEST: string
}

declare const UltimateConfig: ConfigVariables

export default UltimateConfig
```

Thanks!

**Comment by maxkomarychev** [comment]
@tr3v3r I have fixed this in version v1.2.3: https://github.com/maxkomarychev/react-native-ultimate-config/compare/v1.2.2...v1.2.3 please try it out :) thanks!

**Comment by maxkomarychev** [comment]
feel free to reopen if it still exists
